pkg/api/update/update.go

```diff
@@ -43,12 +44,15 @@ func (handle *Handler) Handle(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	select {
-	case chanValue := <-lock:
-		defer func() { lock <- chanValue }()
-		handle.fn()
-	default:
-		log.Debug("Skipped. Another update already running.")
+	var images []string
+	if r.URL.Query().Has("image") {
+		images = strings.Split(r.URL.Query().Get("image"), ",")
```

Comment:
```suggestion
	if image, found := r.URL.Query()["image"] {
		images = strings.Split(image, ",")
```

Reply:
There was a compile error in here, I'm still not sure if this is because my Go version is incorrect, even though I've now explicitly told my IDE to use 1.15.